go/base/management/commands/go_generate_export_conversations_urls.py

```diff
@@ -5,8 +5,7 @@
 from django.core.management.base import CommandError
 from django.utils.text import slugify
 
-from go.base.utils import vumi_api_for_user
-from go.base.command_utils import BaseGoCommand, get_user_by_email
+from go.base.command_utils import BaseGoCommand
 
 
 class Command(BaseGoCommand):
```

Comment:
This probably needs a comment explaining why it's not using `BaseGoAccountCommand`.